hono-router.hbs

```diff
@@ -73,7 +73,7 @@ export function RegisterRoutes<T = any>(router: Hono<any>) {
                 {{/each}}
             };
 
-            let validatedArgs;
+            let validatedArgs: any;
```

Comment:
I'm sure we could do better than `any` but this is no worse than what we have (it was already implicitly `any`) :smile: